src/lib/index.ts

```diff
@@ -0,0 +1 @@
+export const noob = () => {}
```

Comment:
let's change to noop :) (stands for "no operation")
```suggestion
export const noop = () => {}
```